crates/stellar-registry-cli/README.md

```diff
@@ -66,17 +77,53 @@ stellar registry deploy \
 
 2. Install the deployed contract:
 ```bash
-stellar registry install my-token \
-  --out-dir ./contracts/my-token
+stellar registry install my-token
+```
+
+Then can interact with it the contract with `stellar-cli`:
+```bash
+stellar contract invoke --id my-token -- --help
 ```
 
 ## Registry Contract
 
-The registry contract is deployed on testnet at:
+You can install the registry contract like any other:
+```bash
+stellar registry install registry
+```
+
+```
+stellar contract invoke --id registry -- --help
+```
+
+### Transitioning to mainnet
+
+Once you are satisfied with your contract you can publish and deploy on Mainnet.
+
+The first step is adding mainnet to your `stellar-cli`. [See the reccommend list of RPC provides here]( https://developers.stellar.org/docs/data/rpc/rpc-providers)
+
+Then you must add it with the following command:
+```bash
+stellar network add mainnet --network-passphrase "Public Global Stellar Network ; September 2015" --rpc-url <FROM_LIST>
+```
+Then make it your default
+```bash
+stellar network use mainnet
+```
+or if using a `bash` like shell to set it for just the current session:
+```bash
+export STELLAR_NETWORK=mainnet
 ```
-CC2FLEJRHB2Q5JOAJNPFZU25ZAY6IFYXJL7UQ5GF36F3G5QZ4CQILUID
+or if while in the current directory:
+```bash
+echo STELLAR_NETWORK=mainnet >> .env
 ```
 
+## Publishing and then deploying
+
+Publishing and deploing are exactly the same!
```

Comment:
```suggestion
Publishing and deploying are exactly the same!
```

Add a code sample here?

Reply:
Or do you mean "Publishing and then deploying [on Mainnet] is the same as on any other environment"

Reply:
To deploy your contract you'll need to publish first but you can also directly deploy already published contracts. Should I make that clearer?